README.md

```diff
@@ -17,17 +17,20 @@ ROS messages and services used across many ROS packages in the project.
 
 | Topic                  | Type           | Publisher                | Subscriber(s)                               |
 | ---------------------- | -------------- | ------------------------ | ------------------------------------------- |
-| `ais_ships`            | AISShips       | AIS Receiver             | Local Pathfinding                           |
-| `batteries`            | Batteries      | CAN Transceiver          | Local/Remote Transceiver                    |
+| `ais_ships`            | AISShips       | CANTransceiverROSIntf    | Local Pathfinding, Local Transceiver        |
+| `mock_ais_ships`       | AISShips       | Mock AIS                 | CanSimIntf                                  |
+| `batteries`            | Batteries      | CANTransceiverROSIntf    | Local/Remote Transceiver                    |
+| `boat_cmd`             | BoatCmd        | Controller               | CANTransceiverROSIntf                       |
 | `desired_heading`      | DesiredHeading | Local Pathfinding        | Controller, Boat Simulator                  |
-| `data_sensors`         | GenericSensors | CAN Transceiver          | Local/Remote Transceiver                    |
+| `data_sensors`         | GenericSensors | CANTransceiverROSIntf    | Local/Remote Transceiver                    |
 | `global_path`          | Path           | Local/Remote Transceiver | Local Pathfinding                           |
-| `gps`                  | GPS            | CAN Transceiver          | Local/Remote Transceiver, Local Pathfinding |
+| `gps`                  | GPS            | CANTransceiverROSIntf    | Local/Remote Transceiver, Local Pathfinding |
 | `local_path_data`      | LPathData      | Local Pathfinding        | Local/Remote Transceiver                    |
```

Comment:
Can you also update this table to be sorted alphabetically by type? `Path` and `LPathData` are out of order

Reply:
Sorted alphabetically based on type now.